src/shims/fs.rs

```diff
@@ -87,7 +108,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         &mut self,
         fd_op: OpTy<'tcx, Tag>,
         cmd_op: OpTy<'tcx, Tag>,
-        arg_op: Option<OpTy<'tcx, Tag>>,
+        _arg_op: Option<OpTy<'tcx, Tag>>,
```

Comment:
```suggestion
        _arg1_op: Option<OpTy<'tcx, Tag>>,
```
`fnctl` is a varag function, it can have any number of arguments...